spec/mailers/enterprise_mailer_spec.rb

```diff
@@ -14,6 +14,11 @@
       expect(mail.subject)
         .to eq "#{enterprise.name} is now on #{Spree::Config[:site_name]}"
     end
+
+    it "does not set a reply-to email" do
+      EnterpriseMailer.welcome(enterprise).deliver_now
+      expect(ActionMailer::Base.deliveries.first.reply_to).to be nil
+    end
```

Comment:
You can test header fields without delivering the mail. Have a look here:

https://rspec.info/features/6-0/rspec-rails/mailer-specs/mailer-spec/

I'll try to add a commit to show how it could look like.